src/Microsoft.SqlTools.ServiceLayer/Localization/transXliff/sr.de.xlf

```diff
@@ -398,8 +398,8 @@
         <note/>
       </trans-unit>
       <trans-unit id="EE_ExecutionInfo_InitializingLoop">
-        <source>Starting execution loop of {0} times...</source>
-        <target state="new">{0}-malige Batchausführung wurde gestartet.</target>
+        <source>Beginning execution loop</source>
+        <target state="new">Ausführungsschleife wurde {0}-mal gestartet...</target>
```

Comment:
This will cause trouble - we removed the numerical input so the translation needs updating to include this. Is it possible to do `Ausführungsschleife wurde gestartet...` instead?